ynr/apps/people/api/next/filters.py

```diff
@@ -0,0 +1,16 @@
+from django_filters import filterset, filters
+
+from people.models import Person
+
+
+class PeopleFilterMixin(filterset.FilterSet):
```

Comment:
This named using the singular of the model, so `PersonFilter`.

Also a general comment on naming of the filters - the `Mixin` part should be dropped from all of these as they are not really mixins. A mixin is a class that you then inherit in multiple other classes to inherit some shared fields or functions. Wheras these are defined and then used individually as the `filterset_class` on the relevant Viewset

Reply:
Agree on the naming, but I wonder if we _can_ make a mixin for the last updated fields that's shared between all the filters?

Reply:
I did think this - but then I _think_  you would still then need to define the `PersonFilter` that inherits the mixin in order to set the `model` in the Meta? Unless there is a way around this? Or perhaps this would still be better anyway as it saves redefining `last_updated` everywhere

Reply:
Yeah, I think we would need both — happy to leave things as they stand, but we might want to consider if we can/should DRY this up in future. 

One reason I thought about it was in relation to the docs comment — do we need to repeat ourselves in e.g labels for the filter that is used in the docs system?

Reply:
So e.g. you would have:
```
class LastUpdatedFilterMixin(filterset.FilterSet):
    last_updated = filters.IsoDateTimeFilter(
        field_name="modified",
        lookup_expr="gt",
        label="Last updated",
        help_text="An ISO datetime",
    )


class PeopleFilterMixin(LastUpdatedFilterMixin):
    class Meta:
        model = Person
        fields = ["last_updated"]
    
    # repeat for other filters changing the model
```